app/WASMTests.tsx

```diff
@@ -0,0 +1,190 @@
+import { useState, useEffect, FC } from 'react'
+import init from '../pkg/ezkl'
+import {
+  handleGenPkButton,
+  handleGenVkButton,
+  handleGenProofButton,
+  handleVerifyButton,
+  handleGenHashButton,
+} from './Utils'
+
+const filesNames = [
+  'test.onnx',
+  'kzg',
+  'settings.json',
+  'test.provekey',
+  'test.witness.json',
+  'test.proof',
+  'test.key',
+  'message',
+]
+
+interface IFiles {
+  [key: string]: File
+}
+
+interface IResults {
+  [key: string]: string
+}
+
+const TestScript: FC = () => {
+  const [files, setFiles] = useState<IFiles>({})
+  const [results, setResults] = useState<IResults>({})
+
+  useEffect(() => {
+    async function run() {
+      // Initialize the WASM module
+      await init()
+
+      // Load files
+      filesNames.forEach((fileName) => {
+        fetch(`/data/${fileName}`)
+          .then((res) => res.blob())
+          .then((blob) =>
+            setFiles((prevFiles) => ({
+              ...prevFiles,
+              [fileName]: new File([blob], fileName),
+            })),
+          )
+      })
```

Comment:
You're already in an `async` function. Would be more readable to use `await` here